privacyidea/lib/policydecorators.py

```diff
@@ -521,15 +515,11 @@ def login_mode(wrapped_function, *args, **kwds):
                                                           realm=user_object.realm,
                                                           resolver=user_object.resolver,
                                                           user=user_object.login,
-                                                          client=clientip)
+                                                          client=clientip,
+                                                          unique=True)
 
         if login_mode_list:
             # There is a login mode policy
-            if len(login_mode_list) > 1:  # pragma: no cover
-                # We can not decide how to handle the request, so we raise an
-                # exception
-                raise PolicyError(ERROR)
-
             if login_mode_list[0] == LOGINMODE.PRIVACYIDEA:
```

Comment:
I think we can combine this to:

~~~~python
if login_mode_list and login_mode_list[0] == LOGINMODE:PRIVACYIDEA: